### PR TITLE
Capture test result info in JenkinsfileBase

### PR DESCRIPTION
**Pull request by renfeiw**
- set build status
 - visualize test summary

Fixes https://github.com/adoptium/aqa-tests/issues/3864

Signed-off-by: renfeiw <renfeiw@ca.ibm.com>

### CODE REVIEW

**Comment by renfeiw** [comment]
passed: https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/5714/
failed: https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/5716/
no test: https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/5717/

**Review by smlambert — APPROVED** [review]
A valuable enhancement, especially helpful in cases where for whatever reason we are unable to produce TAP files on the test nodes, thanks @renfeiw !

**Comment by renfeiw** [comment]
tested in adoptium: 
https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/5745/
https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/5746/
https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/5747/
https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/5748/

File: buildenv/jenkins/JenkinsfileBase [hunk 1]
```diff
@@ -669,7 +669,50 @@ def runTest( ) {
...
 			}
 		}
+
+		checkTestResults()
+	}
+}
+
+def checkTestResults() {
+	def results = [:]
+	def summary
```
Comment (llxia) [inline buildenv/jenkins/JenkinsfileBase:679]: Can we initialize the value?

**Review by llxia — APPROVED** [review]
Thanks @renfeiw

**Pull request merged**